python/sparktk/models/clustering/gmm.py

```diff
@@ -152,10 +152,10 @@ class GaussianMixtureModel(PropertiesObject):
         >>> assert(np.allclose(expected_sigma, actual_sigma, atol=1e+01))
         </hide>
 
-        >>> model.predict(frame)
+        >>> predicted_frame =  model.predict(frame)
 
         <skip>
-        >>> x = frame.take(9)
+        >>> x = predicted_frame.take(9)
```

Comment:
It might still need the skip, because I've seen before with clustering models, sometimes the cluster numbers can be different (like, what's called cluster 0 could be called cluster 1 and cluster 1 could be called cluster 0).  Even they're the same number of clusters, sometimes they are labelled differently, randomly.